restaurant-api/src/main/java/com/smoothstack/user/service/S3Service.java

```diff
@@ -0,0 +1,76 @@
+package com.smoothstack.user.service;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+
+@Service
+public class S3Service {
+
+    @Value("${csv.bucket}")
+    private String bucketName;
+
+    private static final String[] expectedCsvColumns = {
+            "address",
+            "average_rating",
```

Comment:
Average Rating will not be supplied by the admin, default value will be used until ratings are given by users